src/components/SlateEditor/plugins/image/ImageEmbedForm.tsx

```diff
@@ -77,8 +77,8 @@ const toImageEmbedFormvalues = (embed: ImageEmbedData): ImageEmbedFormValues =>
     lowerRightX: embed.lowerRightX,
     lowerRightY: embed.lowerRightY,
     align: embed.align,
-    size: embed.size?.replace("hide-byline", ""),
-    hideByline: !!embed.size?.includes("hide-byline"),
+    size: embed.size?.replace("--hide-byline", ""),
```

Comment:
Regner med at dette er en konsekvens av migreringen?

Reply:
Ikkje egentlig. Litt rart at det ikkje var slik før, for då har size blitt feks "full--" men det er vel en sjekk på starten på strengen.

Reply:
Full håndteres utenom så det har ikkje hatt nokke å sei før.